scripts/test_translator.py

```diff
@@ -244,11 +244,20 @@ def __init__(self, full_path: str, files: 're.Pattern', keep: List[str], log_lev
         # are broken without it on macOS 12.
         # limit this to macOS because if we do happen to have multiple versions of Clang around, we
         # don't know which to use here, and using the wrong can one break things badly
-        if sys.platform == "darwin":
+        if sys.platform == "darwin" or os.environ.get("C2RUST_USE_NIX") == "1":
```

Comment:
I think you could just add this as a normal argument to `argparse`.  It should be more easily discoverable that way.

Reply:
If this is something automatically set by nix, I think it's fine then.  Could you maybe move this to a `C2RUST_USE_NIX` global set in `main` near the arg parsing, though?  I just want it to be more easily discoverable.  Or something in the `--help` message, too.